crates/broker/src/bin/broker.rs

```diff
@@ -31,8 +34,15 @@ async fn main() -> Result<()> {
     );
     let client = RpcClient::builder().layer(retry_layer).http(args.rpc_url.clone()).boxed();
 
+    let balance_alerts_layer = BalanceAlertLayer::new(BalanceAlertConfig {
+        watch_address: wallet.default_signer().address(),
+        warn_threshold: config.market.balance_warn_threshold.and_then(|s| parse_ether(&s).ok()),
+        error_threshold: config.market.balance_error_threshold.and_then(|s| parse_ether(&s).ok()),
```

Comment:
Silently ignoring the parse failure seems like a footgun. Perhaps at least warn if invalid? This is just on startup so also don't seen an issue with erroring if invalid?

Reply:
Thanks for the reminder. This is fixed in its own branch which also fixes another instance of this same issue https://github.com/boundless-xyz/boundless/pull/411/files